crates/services/p2p/src/config.rs

```diff
@@ -203,6 +206,7 @@ impl Config<NotInitialized> {
             bootstrap_nodes: vec![],
             enable_mdns: false,
             max_peers_connected: 50,
+            max_peers_per_remote_ip: 50,
```

Comment:
```suggestion
            max_peers_per_remote_ip: 5,
```
The config default here doesn't match the default on clap